en/mkdocs.yml

```
@@ -108,6 +108,7 @@ nav:
         - Configure Caching: how-tos/configure-caching.md
         - Manage Microgateway APIs with WSO2 API Manager: how-tos/manage-microgateway-apis-with-wso2-api-manager.md
         - Adding Extension Points: how-tos/adding-extension-points.md
+        - Generating Backend JWT: how-tos/generating-backend-jwt.md
```

Rename the `md` file to match the heading
```suggestion
        - Passing Enduser Attributes to the Backend Using JWT: how-tos/generating-backend-jwt.md
```